### PR TITLE
WIP test cnfdr26

### PR DESCRIPTION
**Pull request by Sandeepyadav93**

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 1]
```diff
@@ -329,3 +329,4 @@ ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -vv || eval $PR
 ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/fetch-information.yml -vv || eval $PROCEED_AFTER_FAILURES
 exit ${status}
 
+sleep 7200
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:332]: `sleep` should be before `exit`

Comment (Sandeepyadav93) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:332]: thanks Sagi!

Comment (Sandeepyadav93) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:331]: Updated, Also updated ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml , timeout was added in SNO job and not in MNO job, corrected.

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Pull request closed without merging**